plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -98,7 +98,7 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
         else -> unimplementedTypeEmbedding(type)
     }
 
-    override fun getField(field: FirPropertySymbol): FieldEmbedding? = fields[field.callableId.embedName()]
+    override fun getField(field: FirPropertySymbol): FieldEmbedding? = fields[field.callableId.embedName(0)]
```

Comment:
This doesn't sit right with me: we end up with a lot of magic zero constants.  Instead, maybe there can be an `atScopeLevel()` function on names that puts them to the right level?